Source/JavaScriptCore/wasm/WasmOSREntryPlan.cpp

```diff
@@ -107,15 +107,15 @@ void OSREntryPlan::work(CompilationEffort)
 
     if (UNLIKELY(!parseAndCompileResult)) {
         Locker locker { m_lock };
-        fail(makeString(parseAndCompileResult.error(), "when trying to tier up ", String::number(m_functionIndex)));
+        fail(makeString(parseAndCompileResult.error(), "when trying to tier up "_s, String::number(m_functionIndex)));
```

Comment:
Should say `m_functionIndex` instead of `String::number(m_functionIndex)`. Doesn’t need to be addressed in this patch, but I probably would.